src/components/BottomSheet/BottomSheet.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import { BottomSheet } from "./BottomSheet";
+import { View, Text } from "react-native";
+
+describe("BottomSheet", () => {
+  it("should render its children properly", () => {
+    expect.assertions(1);
+    const testContent = "content";
+    const component = render(
+      <BottomSheet>
+        {() => (
```

Comment:
How come need a function here?

Reply:
In the typical case, we'll use the `openSheet` argument provided by `BottomSheet`, but since the test doesn't require it, I've omitted it.

I changed the children type of `BottomSheet` so that it also accepts a normal `ReactNode`